examples/exotica_examples/scripts/example_ik_eff_axis_alignment.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+from numpy import array
+from numpy import matrix
```

Comment:
You are not using matrix, hence, remove. In general it is discouraged to use numpy.matrix. Also you can group array under np, i.e., change code from array to np.array (or only include array if you are not using anything else from numpy).

Reply:
This comes from copy/paste. Perhaps `from numpy import matrix` is used in another example file. Either way, this line will be removed.

Reply:
I understand, a lot of the old examples contain garbage for historical reasons. We should clean them up and update API at some point.